src/apps/Script/Script.js

```diff
@@ -212,6 +214,14 @@ const Script = React.createClass({
         flexBasis: 305,
         flexGrow: 1,
         display: 'flex'
+      },
+      linkIconStyles: {
+        color: Colors.blue500,
+        fontSize: 22,
+        lineHeight: 1
+      },
+      resultIconStyles: {
+        padding: '10px 0 0 0'
```

Comment:
`paddingTop: 10` does the same and you don't have to think why there is left/right reset.